media-sound/jack-rack/files/jack-rack-1.4.8_rc1-C23.patch

```diff
@@ -0,0 +1,59 @@
+https://bugs.gentoo.org/946658
+Fix declaration without arguments
+--- a/src/plugin_desc.h
++++ b/src/plugin_desc.h
+@@ -59,7 +59,7 @@
+ plugin_desc_t * plugin_desc_new_with_descriptor (const char * object_file,
+                                                  unsigned long index,
+                                                  const LADSPA_Descriptor * descriptor);
+-void            plugin_desc_destroy ();
++void            plugin_desc_destroy (plugin_desc_t * pd);
+ 
+ void plugin_desc_set_object_file (plugin_desc_t * pd, const char * object_file);
+ void plugin_desc_set_index       (plugin_desc_t * pd, unsigned long index);
+https://bugs.gentoo.org/919502
+compatibility with Lash >= 0.5.2
+--- a/src/ui_callbacks.c	2025-01-15 17:07:29.738203642 +0400
++++ b/src/ui_callbacks.c	2025-01-15 17:07:57.206845017 +0400
+@@ -307,6 +307,41 @@
+ 
+ 
+ #ifdef HAVE_LASH
++
++/* function from LASH, is internal function in 0.5.4, but GPLv2+
```

Comment:
This means we need to make LICENSE "GPL-2+" (or appended at least, with a comment). I'll do that